webportal/webapp/src/app/account/loginsync/togglepassword.directive.ts

```diff
@@ -0,0 +1,35 @@
+import { Directive, ElementRef } from '@angular/core';
+
+@Directive({
```

Comment:
Still getting the lay of the land, let me know if this should live somewhere else.